### ISSUE TITLE
Collections missing in user collection tab

### ISSUE THREAD

**Issue by niklasp** [kodadot/nft-gallery#7056]
### What happened?

I only see 2 collections but have definately more

https://kodadot.xyz/ahk/u/DT7kRjGFvRKxGSx5CPUCA1pazj6gzJ6Db11xmkX4yYSNK7m?tab=collections

e.g. 

https://kodadot.xyz/ahk/collection/101

As you can see [in this gist](https://gist.github.com/niklasp/e07305f7a479c92e9f2f3d2bee4613f2) that is an output from the chain state `nfts.collection.entries()` I am actually the owner of 53 collections ( while some are empty)

### Please reproduce in steps

see above, dont know the reason

### Expected Behavior

See all my collections in the collection tab. Would prefer also to see empty ones for better overview or at least to toggle display of empty

### What browsers are you seeing the problem on?

Chrome, Firefox

### At which address did you encounter bug?

kodadot.xyz

### Are you logged in?

Yes

### Which wallet you are using?

PolkadotJS

### Screenshots

<img width="1377" alt="Bildschirmfoto 2023-08-30 um 21 45 04" src="https://github.com/kodadot/nft-gallery/assets/1685139/84b9b8a3-6d33-4ad3-a3cf-4e0ce24a3c8b">

### Payment link for reward

https://kodadot.xyz/ksm/transfer/?target=DT7kRjGFvRKxGSx5CPUCA1pazj6gzJ6Db11xmkX4yYSNK7m

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by daiagi** [comment]
I agree. this behavior where empty collections are not shown in user profile is annoying

**Comment by roiLeo** [comment]
we already been through this
- #1031 
- #1357
- #1367

**Comment by niklasp** [comment]
@roiLeo ok, but the issue here is that even full collections are not shown (101) above

**Comment by roiLeo** [comment]
> @roiLeo ok, but the issue here is that even full collections are not shown (101) above

Hello, I've check your profile page and it look like this collection is not showing because we filter collection with empty/null metadata

**Comment by niklasp** [comment]
Is that a feature 😊?

**Comment by roiLeo** [comment]
> Is that a feature 😊?

Kinda lol

- ref #6014 & #5974

**Comment by roiLeo** [comment]
Meanwhile: 
- https://canary.kodadot.xyz/ahk/u/DT7kRjGFvRKxGSx5CPUCA1pazj6gzJ6Db11xmkX4yYSNK7m?tab=collections

**Comment by niklasp** [comment]
LoL wut?
